Makefile

```diff
@@ -26,6 +26,11 @@ deps: $(VENV)
 	$(PIP) install lib/kibana
 	$(PIP) install lib/kql
 
+.PHONY: hunting
+hunting: $(VENV) deps
+	@echo "Installing all dependencies..."
+	$(PIP) install elasticsearch~=8.15.1 tabulate==0.9.0
```

Comment:
This is a workaround for 8.12.1 being the version dependency by `detection_rules` and `lib/kibana`.